kover-maven-plugin/src/main/kotlin/kotlinx/kover/maven/plugin/mojo/abstracts/AbstractCoverageTaskMojo.kt

```diff
@@ -226,7 +226,9 @@ abstract class AbstractCoverageTaskMojo : AbstractKoverMojo() {
                 .filter { "compile" in it.goals }
                 .filter { execution -> execution.configuration != null && execution.configuration is Xpp3Dom }
                 .flatMap { execution ->
-                    (execution.configuration as Xpp3Dom).getChild("sourceDirs").children.map { toAbsoluteFile(it.value) }
+                    val config = execution.configuration as Xpp3Dom
+                    val sourceDirs = config.getChild("sourceDirs") ?: return@flatMap emptyList()
+                    sourceDirs.children.map { toAbsoluteFile(it.value) }
```

Comment:
If there are `<sourceDirs><sourceDirs/>`, but no children are specified, will `children` be a null or an emptyList?

Reply:
Empty list, it's not mentioned in the docs, but implementation is
```
    public Xpp3Dom[] getChildren()
    {
        if ( null == childList || childList.isEmpty() )
        {
            return EMPTY_DOM_ARRAY;
        }
        else
        {
            return (Xpp3Dom[]) childList.toArray( new Xpp3Dom[childList.size()] );
        }
    }
```